apps/meteor/client/views/room/Sidepanel/SidepanelItem/useParentTeamData.ts

```diff
@@ -0,0 +1,63 @@
+import { TEAM_TYPE } from '@rocket.chat/core-typings';
+import type { SubscriptionWithRoom } from '@rocket.chat/ui-contexts';
+import { useEndpoint, useUserId } from '@rocket.chat/ui-contexts';
+import { useQuery, keepPreviousData } from '@tanstack/react-query';
+
+import { goToRoomById } from '../../../../lib/utils/goToRoomById';
+
+type APIErrorResult = { success: boolean; error: string };
+
+export const useParentTeamData = (room: SubscriptionWithRoom) => {
+	const { teamId } = room;
+
+	const userId = useUserId();
+
+	if (!teamId) {
+		throw new Error('invalid rid');
+	}
+
+	if (!userId) {
+		throw new Error('invalid uid');
+	}
+
+	const teamsInfoEndpoint = useEndpoint('GET', '/v1/teams.info');
+	const userTeamsListEndpoint = useEndpoint('GET', '/v1/users.listTeams');
+
+	const {
+		data: teamInfoData,
+		isLoading: teamInfoLoading,
+		isError: teamInfoError,
+	} = useQuery({
+		queryKey: ['teamId', teamId],
```

Comment:
I think we need to make these query keys more specific to the endpoint.

Reply:
change to `useTeamInfoQuery`